app/views/spree/admin/reviews/index.html.erb

```diff
@@ -96,7 +96,7 @@
             <% end %>
           </td>
           <td class="align-center">
-            <%= review.user_id ? link_to(review.user.try(:email), [:admin, review.user]) : I18n.t("spree.anonymous") %></p>
+            <%= review.user_id ? link_to(review.user.try(:email), [:admin, review.user]) : I18n.t("spree.anonymous") %><br />
```

Comment:
I'd rather have this as a paragraph I think. Does it cause any issues with the spacing?

Reply:
Sure, it is fixed now. It was missing the opening paragraph markup.